decidim-comments/lib/decidim/comments/api/comment_type.rb

```diff
@@ -51,6 +51,12 @@ module Comments
           obj.down_voted_by?(ctx[:current_user])
         }
       end
+
+      field :hasComments, !types.Boolean, "Check if the commentable has comments" do
```

Comment:
I don't understand: Didn't we have a `hasComments` field before? It looks like it was just added.